web/src/features/charts/ChartTitle.tsx

```diff
@@ -21,13 +22,26 @@ export function ChartTitle({ translationKey }: Props) {
   Use local for timeAverage if exists, otherwise use local default if exists. If no translation exists, use english
   */
   return (
-    <h3 className="text-md">
-      {localExists
-        ? __(`${translationKey}.${timeAverage}`)
-        : __(
-            `${translationKey}.default`,
-            formatTimeRange(localDefaultExists ? i18n.language : 'en', timeAverage)
-          )}
-    </h3>
+    <>
+      <h3 className="pt-3 pb-1 text-md">
+        {localExists
+          ? __(`${translationKey}.${timeAverage}`)
+          : __(
+              `${translationKey}.default`,
+              formatTimeRange(localDefaultExists ? i18n.language : 'en', timeAverage)
+            )}
+      </h3>
+      <div className=" flex flex-row items-center pb-2 text-center text-sm  ">
+        <HiOutlineArrowDownTray className="min-w-[12px]" size={12} />
+        <a
+          href="https://electricitymaps.com/?utm_source=app.electricitymap.org&utm_medium=referral&utm_campaign=country_panel"
```

Comment:
```suggestion
          href="https://electricitymaps.com/?utm_source=app.electricitymaps.com&utm_medium=referral&utm_campaign=country_panel"
```
Shouldn't we use the new domain?